templates/zerver/help/link-to-a-message-or-conversation.md

```diff
@@ -1,10 +1,38 @@
 # Link to a message or conversation
 
-Share permanent links to messages, threads, and streams.  With Zulip,
-it's easy to to link to specific (parts of) conversations from issue
-trackers, documentation, or other external tools.
+Zulip makes it easy to share links to messages, topics, and streams. You can
+link from one Zulip conversation to another, or share links to Zulip conversations
+in issue trackers, emails, or other external tools.
 
-## Link to a stream or topic
+## Link to a stream or topic within Zulip
+
+Zulip automatically creates links to streams and topics in messages you send.
+
+The easiest way to link to a stream or topic is:
+
+{start_tabs}
+
+1. Type `#` followed by the one or more letters of the stream name.
+
+2. Choose the desired stream from the auto-complete menu. The link will be
+   automatically formatted for you.
+
+3. If linking to a topic, type `>` after selecting a stream as described above,
+   followed by one or more letters of the topic name.
+
+4. Choose the desired topic from the auto-complete menu. The link will be
```

Comment:
We might add a connector to the beginning of the second sentence here as it's repeating what was stated above in step 2. Something like, 'Again, the link will be ...'